backend/app/data_layer/database/db_connections/postgresql.py

```
@@ -0,0 +1,57 @@
+from pathlib import Path
+from typing import Generator
+from urllib.parse import quote_plus
+
+from sqlalchemy.engine import Engine
+from sqlmodel import Session, SQLModel, create_engine
+
+from app.data_layer.database.models import *
```

Ruff (0.8.2)

`from app.data_layer.database.models import *` used; unable to detect undefined names

(F403)